app/src/components/Register.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import axios from 'axios'
+
+class Register extends React.Component {
+  state = {
+    username: '',
+    password: ''
+  }
+  inputHandler = e => {
+    let value = e.target.value
+    this.setState({ [e.target.name]: value })
+  }
+  submit = e => {
+    e.preventDefault()
+    const { username, password } = this.state
+    axios
+      .post(`http://localhost:9090/api/register`, {
+        username,
+        password
+      })
+      .then(res1 => {
+        axios
+          .post(`http://localhost:9090/api/login`, {
+            username,
+            password
+          })
+          .then(res => {
+            this.props.setToken(res.data.token)
+          })
+          .catch(err => console.log(err))
+      })
+      .catch(err => console.log(err))
```

Comment:
You could skip the extra steps here and just use console.log for your callbacks where you're just passing the parameter along:
```javascript
    // ...
    .catch(console.log)
  })
  .catch(console.log)
```